### DISCUSSION TITLE
Lag spikes in gameplay since 2023.511 update

### DISCUSSION THREAD

**Discussion by rvnexe** [ppy/osu#23472] [Q&A]
### Type

Game behaviour

### Bug description

I have several and continous lag spikes in gameplay since today's update. I didn't had this lag before.

### Version

2023.511.0-lazer

### Logs

[performance.log](https://github.com/ppy/osu/files/11451000/performance.log)
[runtime.log](https://github.com/ppy/osu/files/11451001/runtime.log)
[updater.log](https://github.com/ppy/osu/files/11451002/updater.log)
[database.log](https://github.com/ppy/osu/files/11451003/database.log)
[input.log](https://github.com/ppy/osu/files/11451004/input.log)
[legacy-ipc.log](https://github.com/ppy/osu/files/11451005/legacy-ipc.log)
[network.log](https://github.com/ppy/osu/files/11451006/network.log)

**Comment by rvnexe** [comment]
Edit: I have tested it with both D3D11 and OpenGL

**Comment by rvnexe** [comment]
Update: The Lag fixes when I set window mode to windowed (was exclusive fs)

**Comment by rvnexe** [comment]
Update: My GPU and CPU usages are not 100%. Idk why. Framerate is unlimited too in osu! settings.
![image](https://github.com/ppy/osu/assets/89677246/953af838-ec4e-4277-a966-71505411ba7d)

**Comment by Theighlin** [comment]
This is not a chat, if you want to post updates, edit the opening post until someone responds.
